dashboard/app/models/user.rb

```diff
@@ -840,6 +845,8 @@ def self.find_for_authentication(tainted_conditions)
     login = conditions.delete(:login)
     if login.present?
       return nil if login.utf8mb4?
+      # TODO: multi-auth (@eric, before merge!) have to handle this path, and make sure that whatever
```

Comment:
Still TODO, I'm having trouble figuring out what the right thing to do here is - logged as a work item.